### ISSUE TITLE
Error building wheel for installation on windows 11

### ISSUE THREAD

**Issue by Chiken502** [belangeo/pyo#282]
Hello, I'm relatively new to Python and unsure if I've made a mistake, but I'm seeking a resolution for an issue so I can explore this library. The problem arises when using `pip install pyo`.
````
      copying pyo\examples\snds\snd_6.aif -> build\lib.win-amd64-cpython-312\pyo\examples\snds
      copying pyo\examples\snds\mapleleafrag.mid -> build\lib.win-amd64-cpython-312\pyo\examples\snds
      running build_ext
      error: [WinError 2] The system cannot find the file specified
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyo
  Running setup.py clean for pyo
Failed to build pyo
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (pyo)
````

The issue doesn't seem to be with pip, so I attempted a manual download from GitHub. In my code, I used the import statement `from pyo_master import *`, where 'pyo_master' was the name of the downloaded file. This approach appeared to work with the existing example code, but upon execution, it resulted in another error which is below.
````
....\pyo\lib\_core.py", line 48, in <module>
    from .._pyo import *
ModuleNotFoundError: No module named 'pyo._pyo'
````

I don't know how to fix this issue, so I hope someone here can help. Below is additional information that might be useful in addressing the issue.
Operating System - Windows 11
Python Version - 3.12.3

**Comment by alexdrymonitis** [comment]
This usually happens when you run your Python script in the directory of the source code of Pyo. Try running your Python code from another directory.

**Comment by stripwax** [comment]
But I also think the original question still stands which is the error with "pip install pyo" - the other items were related to attempting workaround that initial failure.

**Comment by alexdrymonitis** [comment]
Afaik, Pyo runs in Python3.11 max. That could be the issue. Check here https://belangeo.github.io/pyo/download.html

**Comment by belangeo** [comment]
Yes, there are no wheels yet for 3.12 and 3.13, so pip has to build pyo from sources. We don't see the error here, but I suspect that the build system didn't find pyo's dependencies. Next release will have 3.12 and 3.13 wheels on all platforms (I already have it building on Windows). I'll closed this issue as it is not common to build pyo from sources on Windows. Wheels coming soon.
